src/pages/DiaryList/DiaryList.jsx

```diff
@@ -1,37 +1,13 @@
-import React, { useState } from "react";
+import React from "react";
 import DiaryCard from "../DiaryCard/DiaryCard";
 import styles from "../DiaryList/DiaryList.module.css";
-import EditModeDiaryCard from "../../components/EditModeDiaryCard/EditModeDiaryCard";
-
-export default function DiaryList({ diaries, onDelete, onSave }) {
-  const [editingDiaryId, setEditingDiaryId] = useState(null);
-
-  const handleEdit = (id) => {
-    setEditingDiaryId(id);
-  };
-
-  const handleSave = (updatedDiary) => {
-    onSave(updatedDiary);
-    setEditingDiaryId(null);
-  };
-
-  const cancelEditMode = () => {
-    setEditingDiaryId(null);
-  };
 
+export default function DiaryList({ diaries, onDelete }) {
```

Comment:
you don't need `onDelete` prop

Reply:
deleted onDeleted Prop

Reply:
resolved by https://github.com/Gracekangdh/DiaryApp/pull/1/commits/2120610c15e9fbd68d98ef37b1ba1811a24f50f6